src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -729,9 +737,9 @@ public void serializeToView(@NotNull final PacketBuffer buf)
         buf.writeInt(getPrimarySkill().ordinal());
         buf.writeInt(getSecondarySkill().ordinal());
         buf.writeInt(getMaxInhabitants());
-        buf.writeBoolean(hasSpaceForMoreRecipes());
         buf.writeBoolean(isRecipeAlterationAllowed());
         buf.writeString(jobDisplayName);
+        buf.writeInt(maxRecipes());
     }
```

Comment:
Much better to synch the whole number instead of just "is there more space"

Reply:
I must admit I was worried when I changed this at first, removing something from a buffer always seems a fragile point to me